buckets-mdapi/src/sql.rs

```diff
@@ -191,3 +180,16 @@ fn post_timer_metrics<T>(
         .with_label_values(&[&method.as_str(), success])
         .observe(t);
 }
+
+pub fn get_sql(vnode: u64) -> String {
```

Comment:
I might suggest relocating this to the `object` module. That's already the home of some other object-specific SQL and it provides the context that it is related specifically to objects and not buckets as well. This module is a bit of misnomer perhaps. It's more of the supporting functions for issuing queries versus the queries themselves.

Reply:
I can agree with this. I've moved it to object instead.